### PR TITLE
Add page on Julia community stewards

### PR DESCRIPTION
**Pull request by timholy**
I'm not certain whether the link I inserted will work, but one can always hope.

### CODE REVIEW

File: community/standards/julia_stewards.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Julia Community Policy Regarding Community-Standards Violations
```
Comment (stevengj) [inline community/standards/julia_stewards.md]: link `Community-Standards` to the standards page (probably also below).

File: community/standards/julia_stewards.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+As needed, this group can be contacted at [stewards@julialang.org](mailto:stewards@julialang.org).
+Other than the committee listed above, there are no other recipients of emails sent to this address, and **all communications shall be treated confidentially**.
+
+
+The guidelines governing the Julia Stewards' actions are described below.
+
+## Complaint Resolution
+
+**No one shall be retaliated against for good-faith participation in a complaint**.
+In general, violations of the community standards will generate one or more of the following responses:
```
Comment (stevengj) [inline community/standards/julia_stewards.md]: responses from the Stewards

File: community/standards/julia_stewards.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   * A consensus committee of 3+ disinterested stewards (possibly including non-steward members) shall investigate the complaint, communicating this with both the complainant (if any) and the respondent (alleged violator).
+   * The investigation shall be as confidential as possible, and this expectation shall be communicated to all participants. Identities and testimonies of participants shall, where possible, be known only to the committee.
+   * The committee shall provide a private written report detailing its findings and recommendation to the stewards, complainant, and respondent. If a violation has been found, recommendations may include bans and other alterations of online privileges, or in less severe cases may request apologies and other informal resolutions.
+   * The conclusions of the investigative committee are final and cannot be appealed except in cases of gross misconduct or major factual errors.
+
+## Bans
+
+The public Julia mailing lists (e.g. julia-users) and GitHub pages (e.g. issues and pull requests) are normally open forums for all good-faith contributors.
```
Comment (stevengj) [inline community/standards/julia_stewards.md]: format `julia-users`

**Comment by StefanKarpinski** [comment]
How about moving `community/standards/julia_stewards.md` file to `community/stewards.md`?

File: community/standards/julia_stewards.md (hunk 3 above)
Comment (StefanKarpinski) [inline community/standards/julia_stewards.md]: Better still, make it a link to the google group: https://groups.google.com/forum/#!forum/julia-users.

File: community/standards/julia_stewards.md (hunk 2 above)
Comment (garrison) [inline community/standards/julia_stewards.md]: s/Katherine/Katharine/;

**Comment by simonbyrne** [comment]
The links and pages now work, though I didn't address the other comments.

**Comment by stevengj** [comment]
Argh, all the other comments are now hidden as referring to an "outdated diff" due to the rename.

**Comment by stevengj** [comment]
Okay, I addressed the other comments so far.

**Comment by timholy** [comment]
Thanks, everyone. I'll give this a couple more hours, then announce this evening.

File: community/stewards/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+## Bans
+
+The public Julia mailing lists (e.g. [julia-users](https://groups.google.com/forum/#!forum/julia-users)) and GitHub pages (e.g. issues and pull requests) are normally open forums for all good-faith contributors.
+In extreme cases, repeated violations of the Community Standards may lead to the banning of a contributor from one or more of these public forums.
+Such total bans are expected to be rare and are not to be undertaken lightly.
+
+
+Note that:
+* Long-term bans shall only be undertaken in response to the recommendations of a formal investigation by the Julia stewards as described above. (A temporary ban may be imposed by Julia admins in response to an urgent situation, but shall be followed by a formal investigation if the ban is to continue.)
+* After a predetermined period, not to exceed six months, the banned individual may request that the ban be lifted, by acknowledging his/her past violations and providing a written plan to avoid such violations in the future. This request will be typically be considered by the same committee that recommended the initial ban.
```
Comment (stevengj) [inline community/stewards/index.md]: For some reason, this isn't showing up as a bulleted list?

Comment (timholy) [inline community/stewards/index.md]: Fixed.

**Comment by stevengj** [comment]
Should this be announced on julia-users?

**Comment by ViralBShah** [comment]
Let's announce

**Comment by timholy** [comment]
Already done (https://groups.google.com/d/msg/julia-dev/-MGVkWq-JUU/WSFqMnOZCQAJ and https://groups.google.com/d/msg/julia-users/AiB72jXnFJ4/AabEBWGnCAAJ).

File: community/stewards/index.md (hunk 4 above)
Comment (rfourquet) [inline community/stewards/index.md]: remove one "be" in "be typically be"

Comment (stevengj) [inline community/stewards/index.md]: Thanks, fixed in fb261452ce25ca442cd98d35b6f02848a52699f2

**Pull request merged**